apps/web/src/views/Info/components/InfoTables/TransactionsTable.tsx

```diff
@@ -109,7 +110,10 @@ const DataRow: React.FC<React.PropsWithChildren<{ transaction: Transaction }>> =
 
   return (
     <ResponsiveGrid>
-      <LinkExternal isBscScan href={getBlockExploreLink(transaction.hash, 'transaction', multiChainId[chainName])}>
+      <LinkExternal
+        isBscScan={multiChainId[chainName] === ChainId.BSC}
```

Comment:
Can we refactor the `LinkExternal` component as well? We're gonna support multiple chains so better not be chain specific.

There should be a `ScanLink` component and `chainId` be one of the props